content/english/java-tictactoe/_index.md

```diff
@@ -9,9 +9,22 @@ draft: false
 ---
 
 ## Overview
+In this workshop, you will learn how to create a Tic-Tac-Toe program in Java from start to finish! 
 
+You will need basic Java knowledge from the workshop [Java: Basics](../../java-basics/_index.md) as we will assume you to know about variable, operators, classes, and arrays during the exercises.
```

Comment:
Is there a java basics workshop already or the plan is to move basics out of this workshop later?

Reply:
Oh just saw that! We have them already, great job! 💯